nightingale_interface/screens/screen_config.py

```diff
@@ -0,0 +1,22 @@
+class ScreenConfig:
```

Comment:
Now that I know the main class is basically global using "self.", can probably remove this and replace

Reply:
agreed, this is still helpful if we are using enums though